packages/gestalt/src/SearchGuide.tsx

```diff
@@ -141,7 +146,16 @@ const SearchGuideWithForwardRef = forwardRef<HTMLButtonElement, Props>(function
           </Box>
         )}
         {'image' in thumbnail && (
-          <div className={isInVRExperiment ? styles.imageDivVr : styles.imageDiv}>
+          <div
+            className={
+              isInVRExperiment
+                ? classnames({
+                    [styles.imageDivVr]: true,
```

Comment:
should we do here:
[styles.imageDivVr]: !selected,
[styles.selectedImageDivVr]: selected,

Reply:
fixed, selectedImageDiv shouldn't be there anymore